### ISSUE TITLE
Include full submission context in Game Screen feedback?

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/balancing-chemical-equations#238]
**Test device**
Asus Zenbook

**Operating System**
Win 11

**Browser**
Firefox

**Problem description**
From https://github.com/phetsims/qa/issues/1361

Should answer feedback on the Game Screen include the full submission information (e.g., “Balanced,” “Not Simplified,” or “Not Balanced”) to provide clearer context?

**Visuals**

<img width="1564" height="636" alt="Image" src="https://github.com/user-attachments/assets/364e12c0-3eaf-4b7b-99b7-f6fd2815ab57" />

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Balancing Chemical Equations‬
URL: https://phet-dev.colorado.edu/html/balancing-chemical-equations/2.1.0-rc.3/phet/balancing-chemical-equations_all_phet.html?postMessageOnLoad=&postMessageOnError=&supportsInteractiveDescription=true
Version: 2.1.0-rc.3 2026-02-24 20:58:59 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:148.0) Gecko/20100101 Firefox/148.0
Language: es-ES
Window: 1280x587
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (ANGLE (AMD, Radeon HD 3200 Graphics Direct3D11 vs_5_0 ps_5_0), or similar)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>
